### PR TITLE
Explain the default value of arg_separator

### PR DESCRIPTION
**Pull request by kamil-tekiela**

### CODE REVIEW

File: reference/url/functions/http-build-query.xml [hunk 1]
```diff
@@ -62,7 +62,8 @@
       <para>
        <link linkend="ini.arg-separator.output">arg_separator.output</link>
        is used to separate arguments but may be overridden by specifying
-       this parameter.
+       this parameter. If <parameter>arg_separator</parameter> is &null; and the
+       INI directive is not set, the default separator is <literal>&amp;</literal>.
```
Comment (cmb69) [inline reference/url/functions/http-build-query.xml:66]: Right, but `$arg_separator` is only nullable as of PHP 8.0.0, but the respective changelog entry is missing. I'm not sure whether we should cater to that ("is &null; or omitted").

Comment (afilina) [reply reference/url/functions/http-build-query.xml:66]: Pinging author @kamil-tekiela to address review comment.

**Comment by kamil-tekiela** [comment]
Closing because I can't remember what this PR was about.

**Comment by afilina** [comment]
Understandable :)

**Pull request closed without merging**